tests/test_sampler.py

```diff
@@ -148,13 +152,22 @@ def test_sampler_iterator(backend_name):
         assert rl[i]["iteration"] == iteration_list[i]
     # Test that the results changes given the iteration parameters (avoid Clifford as sampling adds randomness)
     if backend_name == "SLOS":
-        assert rl[0]["results"][pcvl.BasicState([1, 1])] == pytest.approx(0.7701511529340699)
```

Comment:
Did you remove these assert statements (L151) because it tests very precise value for probs? or because it failed? 

Maybe you want to keep it and set a tolerance by

'output_value = pytest.approx(target_value, abs=+/-tolerance)'

Reply:
It was because the processor changed so did the output